pkgs/tools/package-management/rpm/default.nix

```diff
@@ -34,13 +37,33 @@ stdenv.mkDerivation rec {
     hash = "sha256-N/O0LAlmlB4q0/EP3jY5gkplkdBxl7qP0IacoHeeH1Y=";
   };
 
+  patches = [
+    # Resolves `error: expected expression` on clang
+    # See: https://github.com/rpm-software-management/rpm/issues/2435.
+    (fetchpatch2 {
+      url = "https://github.com/rpm-software-management/rpm/commit/3b7cc5b5362fc22cb6f96180741e2701e703844d.diff?full_index=1";
+      hash = "sha256-pBKYQK97RkK9bigroNVgeDrR37poFSUBdEpB/RjwK5A=";
+    })
```

Comment:
This is included in 4.19 and 4.20; we should probably bump the version instead. If not, we should at least point to `b960c0b43a080287a7c13533eeb2d9f288db1414`, which is the version of the commit that was landed and won’t get GC’d.